read-fonts/src/collections/int_set/bitpage.rs

```diff
@@ -97,18 +96,23 @@ impl BitPage {
     pub(crate) fn insert(&mut self, val: u32) -> bool {
         let ret = !self.contains(val);
```

Comment:
Was looking at what contains() does and it also grabs the element and applies a bit mask, I wonder if it might be better to just check element_mut & bit_mask inside this function prior to applying it.